secure/storage/github/src/lib.rs

```diff
@@ -178,15 +178,21 @@ impl Client {
             .set("Authorization", &format!("token {}", self.token))
             .set(ACCEPT_HEADER, ACCEPT_VALUE)
             .timeout_connect(TIMEOUT);
-        match env::var("https_proxy") {
-            Ok(proxy) => request
-                .set_proxy(
-                    ureq::Proxy::new(proxy)
-                        .expect("Unable to parse https_proxy environment variable"),
-                )
-                .build(),
-            Err(_e) => request,
+
+        let proxy = Proxy::new();
+        let host = request.get_host().expect("unable to get the host");
+        let scheme = request
```

Comment:
Hmm... is there a default scheme? What happens if I just pass in www.google.com, I'm concerned this might now be broken. maybe we can clean up how we do the match statement Some("http") ... _ => ...

Reply:
I'll revert he functionality change so that it matches whats currently in master